src/adagrams.js

```diff
@@ -1,8 +1,72 @@
+
+
 const Adagrams = {
+  letterArray: ["A", "A", "A", "A", "A", "A", "A", "A", "A",
+    "B", "B", "C", "C", "D", "D", "D", "D", "E", "E", "E", "E", "E",
+    "E", "E", "E", "E", "E", "E", "E", "F", "F", "G", "G", "G", "H",
+    "H", "I", "I", "I", "I", "I", "I", "I", "I", "I", "J", "K", "L",
```

Comment:
This data structure works, but it's not very DRY and would be tricky to change. For example, suppose I said you had the wrong number of "I"s - you would have to do a lot of counting to solve the problem.

Instead you might store a hash of letter frequencies like this

```
letterQuantities: {
  A: 9, B: 2, ...
}
```

and use it to build a pool of letters dynamically.